rke/templates/flannel_v0.14.0Rancher2.go

```diff
@@ -0,0 +1,284 @@
+package templates
```

Comment:
Please rename the file to `flannel_v0.14.0-rancher2.go` to follow the established naming convention, see https://github.com/rancher/kontainer-driver-metadata/tree/dev-v2.7/rke/templates

Reply:
updated